### DISCUSSION TITLE
actions/setup-python patch number

### DISCUSSION THREAD

**Discussion by Majdoddin** [community/community#63775] [Actions] [answered]
### Select Topic Area

Question

### Body

Which patch number does get installed? is it 3.10.0 or 3.10.12?

steps:
- uses: actions/checkout@v3
- uses: actions/setup-python@v4
  with:
    python-version: '3.10' 
- run: python my_script.py

P.S. 
```<!--- Please direct any generic questions related to actions to our support community forum at https://github.community/c/code-to-cloud/github-actions/41 --->```
This message shows up when creating an issue in actions/setup-python, but the url does not exist anymore.

**Comment by EnriquezAlberto** [comment, marked as answer]
the actual patch number that gets installed may change over time as new releases and updates are made to Python 3.10. It's possible that the version installed could be 3.10.0 or a later patch version like 3.10.12 depending on what has been released at the time you run the workflow.

Regarding the URL issue you mentioned for actions/setup-python, if the URL referenced in the error message does not exist anymore, it could indicate that there has been a change or update to the setup process for Python in GitHub Actions. You might want to refer to the latest documentation or resources related to setting up Python in GitHub Actions to ensure you have accurate and up-to-date information.

For the most current and accurate information about Python versions and setup in GitHub Actions, please refer to the GitHub Actions documentation and the repository for the actions/setup-python action.

**Comment by Majdoddin** [comment]
Thank you for your answer. 
So does it mean that always the _latest_ released patch gets installed?

regarding the url, please see the updated question.
